docs/concepts/trading-on-vega/discounts-rewards.md

```diff
@@ -65,6 +65,14 @@ All of the details for the volume discount program are proposed and accepted thr
 * [Spec: Technical design of the volume discount program ↗](https://github.com/vegaprotocol/specs/blob/master/protocol/0084-VDPR-volume_discount_program.md).
 :::
 
+### Rewards for eligible participants
+Participants who exceed certain eligibility criteria can be rewarded when the eligible entities reward is funded. 
+
+When setting up the reward, one or both criteria can be set:
+
+* Minimum number of VEGA tokens that need to be staked
+* Minimum notional time-weighted average position, on any market using the asset nominated in the reward, or the specified market/markets
```

Comment:
We technically also would allow neither to be set, if a markets scope is specified.

It's use-case would be reward anyone trading on one of these markets, niche and game-able so maybe not worth mentioning.